### PR TITLE
Fix bugs from dev

### PR DESCRIPTION
**Pull request by fuentesloic**
- [x] avatar on all page = _due to onerror missing and css erased by merge_
- [x] default image on article = _due to image missing, erasing by db creation_
- [x] Leaflet controler broke = _due to mixed active content blocked_
- [x] Moving on user = _due to move to user missing function_

### CODE REVIEW

File: templates/blog/macros.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <a href={{ url }} lang={{ article.language }}>
       <h1>{{ article.title|title }}</h1>
       <figure>
-        <img src="{{ article.image | default('/static/uploads/articles/default.png', true) }}" width=100%>
+        <img src="{{ article.image | default('../static/images/article.png', true) }}" width=100%>
```
Comment (vinyll) [inline templates/blog/macros.html]: `../static/` does not make sense. `/static/` is at the root url already.
However, `url_for('static', filename='images/article.png')` would take all its sense ;)

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/blog/macros.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <header class=meta>
       {% if with_author %}
       <a href={{ url_for_user(article.author) }}>
-        <img class=avatar-img src={{ article.author.avatar }} width=40 height=40>
+        <img src={{ article.author.avatar }} width=40 height=40 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
```
Comment (newick) [inline templates/blog/macros.html]: need `alt=""

File: templates/blog/macros.html (hunk 1 above)
Comment (newick) [inline templates/blog/macros.html]: need `alt=""

File: templates/home.html [hunk 3]
```diff
@@ -74,7 +74,7 @@
       <li>
         <a href='{{ url_for_user(user) }}'>
           <figure>
-            <img class=avatar-img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+            <img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
```
Comment (newick) [inline templates/home.html]: need `alt=""

File: templates/layout.html [hunk 4]
```diff
@@ -20,7 +20,7 @@
     <a href={{ url_for('around') }}>Around me</a>
     {% if current_user and current_user.is_authenticated %}
     <div id=avatar-menu class=avatar>
-      <img src={{ current_user.avatar }} class=avatar-img width=35 height=35>
+      <img src={{ current_user.avatar }} width=35 height=35 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
```
Comment (newick) [inline templates/layout.html]: guess what, need `alt="" :D

File: templates/user/profile.html [hunk 5]
```diff
@@ -19,7 +19,7 @@
 <header>
   <div class="profile-intro wrapper">
     <figure>
-      <img class=avatar-img name=avatar src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+      <img name=avatar src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
```
Comment (newick) [inline templates/user/profile.html]: do I need to say? 😄

**Comment by newick** [comment]
capture d ecran 2017-09-08 a 16 50 00
When you write an article, avatar is broken (it'll be changed with styleguide but if we merge dev with master, it'll be ugly)

**Comment by fuentesloic** [comment]
@newick @vinyll done!!  (from my eyes)

File: .gitignore [hunk 6]
```diff
@@ -5,3 +5,4 @@ __pycache__/
 /static/uploads/*
 /node_modules/
 /static/components/App.js
+/static/dist
```
Comment (vinyll) [inline .gitignore]: I don't think this should be gitignored

**Review by vinyll — APPROVED** [review]

**Pull request merged**